apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,326 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { Field, Wizard, WizardPage } from "@wso2is/form";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+    HyprAuthenticationProviderCreateWizardFormValuesInterface
+} from "./hypr-authentication-provider-create-wizard";
+import { getIdentityProviderList } from "../../../api";
+import { IdentityProviderManagementConstants } from "../../../constants";
+import { IdentityProviderListResponseInterface, IdentityProviderTemplateInterface } from "../../../models";
+import { handleGetIDPListCallError } from "../../utils";
+
+/**
+ * Proptypes for the HyprAuthenticationWizardFrom.
+ */
+interface HyprAuthenticationProviderCreateWizardContentPropsInterface extends TestableComponentInterface {
+    /**
+     * Trigger form submit.
+     * @param submitFunctionCb - Callback.
+     */
+    triggerSubmission: (submitFunctionCb: () => void) => void;
```

Comment:
The param name `submitFunctionCb` feels ambiguous IMO. Shall we rename as follows?

```suggestion
    triggerSubmission: (submitFunctionCallback: () => void) => void;
```

Reply:
addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)